source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_cfg.py

```diff
@@ -165,6 +159,9 @@ class SimulationCfg:
     physics_prim_path: str = "/physicsScene"
     """The prim path where the USD PhysicsScene is created. Default is "/physicsScene"."""
 
+    device: str = "cuda:0"
+    """The device for running the simulation/environment. Default is ``"cuda:0"``."""
```

Comment:
Just so I'm understanding correctly - the device is (and was previously) extracted from the `AppLauncher` config for GPU-based workflow, right? If not, likely need to modify my suggestion below.

Might add the valid options here to help new users:
```suggestion
    """The device for running the simulation/environment. Valid options are one of 
    - ``"cpu"``: Use CPU
    - ``"cuda"``: Use GPU - where the device ID of the GPU to run on is inferred from :class:`AppLauncher`'s config. 
    - ``"cuda:N"`` Use GPU where N is the device ID associated with the GPU to run on.
    
    Default is ``"cuda:0"``."""
```